lib/atom-music-playlist-view.js

```diff
@@ -0,0 +1,74 @@
+/** @babel */
+
+/** @jsx etch.dom */
+
+const etch = require("etch");
+const SelectList = require("atom-select-list");
+
+class PlayListView {
```

Comment:
@UziTech I think this class needs some refactoring.
For example the name `PlayListView` is misleading, i think.
Maybe something like `SearchView` would be better in my opinion.
And some parts of the class can become optimized.
What do you think about it?

Reply:
@UziTech Okay, do you want to do these changes in this PR or should we do it later?